weave-js/src/common/components/elements/ModifiedDropdown.tsx

```diff
@@ -470,6 +471,15 @@ const ModifiedDropdown: FC<ModifiedDropdownProps> = React.memo(
           }
         }}
         trigger={renderTrigger()}
+        icon={
+          passProps.className === 'runs-selector-modified-dropdown' ? (
```

Comment:
`runs-selector` seems like logic that shouldn't be in this layer. Maybe add a `useIcon` prop?

Reply:
Sounds good -- added a `useIcon` prop